README.md

```diff
@@ -34,21 +32,18 @@ In order to use this plugin you need "perf" to be installed on a Linux target ho
 
 ### System Requirements
 
-* root privileges
-* "perf" installed on a host
-* /proc/sys/kernel/perf_event_paranoid set to 0 (in other case list of metrics can be limited)
-* Linux kernel version at least 2.6.31
-* [golang 1.5+](https://golang.org/dl/) (needed only for building)
-
-
+- Linux (kernel 2.6.31+)
+- root privileges
+- "perf" installed on a host
+- /proc/sys/kernel/perf_event_paranoid set to 0 (in other case list of metrics can be limited)
 
 ### Installation
 
 #### Download perfevents plugin binary:
-You can get the pre-built binaries for your OS and architecture at snap's [GitHub Releases](https://github.com/intelsdi-x/snap/releases) page.
+You can get the pre-built binaries for your OS and architecture at plugins's [GitHub Releases](https://github.com/intelsdi-x/snap-plugin-collector-perfevents/releases) page. Download the plugin from the latest release and load it into snapd (`/opt/snap/plugins` is the default location for Snap packages).
 
 #### To build the plugin binary:
-Fork https://github.com/intelsdi-x/snap-plugin-collector-perfevents
+Use https://github.com/intelsdi-x/snap-plugin-collector-perfevents or your fork as repo.
```

Comment:
Please add a newline after this sentence, with the newline it will look better on the website ;)